src/components/user/SignIn.js

```diff
@@ -1,65 +1,196 @@
+  const navigateToRegistration = () => {
+    window.location.href = '/register'; // URL of your registration page
+  };
+
+  // Main container styling with maroon/purple background color
+  const mainContainerStyle = {
+    width: '100%',
+    minHeight: '100vh',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    overflow: 'auto',
+    backgroundColor: 'rgba(91, 46, 72, 1)', // Background color for entire screen
+  };
+
+  const dontHaveAccountTextStyle = (fontSize, fontWeight, marginBottom, marginTop, color = 'rgba(255, 255, 255, 1)') => ({
+    color: color,
+    textAlign: 'center',
+    lineHeight: 'normal',
+    fontFamily: 'Montserrat',
+    fontSize: fontSize,
+    fontWeight: fontWeight,
+    textDecoration: 'none',
+    marginBottom: marginBottom,
+    marginTop: '60px', // Added marginTop for more control over spacing
+  });
+
+  // Adjusted sign in box styling for vertical coverage
+  const signInBoxStyle = {
+    maxWidth: '600px',
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around', // Ensures even distribution of space
+    alignItems: 'center',
+    position: 'relative',
+    overflow: 'hidden',
+    paddingTop: '20px', // Padding at the top
+    paddingBottom: '20px', // Padding at the bottom
+  };
+
+  // Adjusted text styling function for minimal spacing
+  const textStyle = (fontSize, fontWeight, marginBottom, color = 'rgba(255, 255, 255, 1)') => ({
+    color: color,
+    textAlign: 'center',
+    lineHeight: 'normal',
+    fontFamily: 'Montserrat',
+    fontSize: fontSize,
+    fontWeight: fontWeight,
+    textDecoration: 'none',
+    marginBottom: marginBottom,
+    marginTop: '20px',
+  });
+
+  // Separate style for the "Sign In with Google" button
+  const googleSignInButtonStyle = {
+    backgroundColor: '#ffffff',
+    width: '300px',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: '1px',
+    borderRadius: '30px',
+    marginBottom: '15px',
+    marginTop: '70px',
+    boxShadow: '0px 4px 8px rgba(0, 0, 0, 0.8)',
+  };
+
+  // Adjusted function for login button styling
+  const loginButtonStyle = () => ({
+    // Existing login button styles
+    backgroundColor: 'rgba(91, 46, 72, 1)',
+    width: '300px',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: '20px',
+    borderRadius: '20px',
+    boxShadow: '0px 4px 8px rgba(0, 0, 0, 1)',
+    marginBottom: '10px', // Reduced spacing to minimize empty space
+    marginTop: '10px',
+  });
+
+  // Separate style for the sign-up button
+  const signUpButtonStyle = {
+    backgroundColor: '#ffffff',
+    width: '300px',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: '10px',
+    borderRadius: '40px',
+    marginBottom: '40px',
+    marginTop: '1px',
+    boxShadow: '0px 4px 8px rgba(0, 0, 0, 0.8)',
```

Comment:
Same issue that was present in the last review of pull request on ffffff is white and pizel-specific stuff